### ISSUE TITLE
Windows binary

### ISSUE THREAD

**Issue by sergeevabc** [vova616/xxhash#1]
Dear @vova616,
Could you be so kind to generate ```.exe``` for the rest of us who are mere Windows users w/o compiler?

**Comment by vova616** [comment]
windows exe with an input of a file?

**Comment by sergeevabc** [comment]
@vova616, yes, execution is expected pretty much as [XXHash][1] by @Cyan4973 works. 

[1]: https://github.com/Cyan4973/xxHash
